setup.py~

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
```

Comment:
I am really sorry. I plan to remove all those backup files and write correct commit message once I get the correct solution. These are just for fast git push to github. I'll squash all the commits once I'm done.